external-crates/move/crates/move-compiler/tests/custom_rules/constant_naming.move

```diff
@@ -0,0 +1,20 @@
+module 0x42::M {
```

Comment:
Could make the file something like `external-crates/move/crates/move-compiler/tests/linter/constant_naming.move`

The change here is changing the dir `custom_rules` to `linter`

Reply:
refactored